plugins/gstreamer/src/Test.cpp

```diff
@@ -22,7 +22,8 @@
 
 int main()
 {
-	char pipelineVariable[] = "NEURALA_GSTREAMER_PIPELINE=playbin uri=https://www.freedesktop.org/software/gstreamer-sdk/data/media/sintel_trailer-480p.webm ! video/x-raw,format=BGR ! videoconvert ! appsink name=mysink";
+	// video-sink="videoconvert ! video/x-raw,width=320,height=240 ! ximagesink"
+	char pipelineVariable[] = "NEURALA_GSTREAMER_PIPELINE=playbin uri=https://www.freedesktop.org/software/gstreamer-sdk/data/media/sintel_trailer-480p.webm video-sink=\"videoconvert ! video/x-raw,width=320,height=240 ! appsink name=mysink\"";
```

Comment:
`mysink` is the default name?

Reply:
Can be anything you want. The name is passed into the plugin via `NEURALA_GSTREAMER_SINK`. I was hoping to use *just* that, without needing to involve that `video-sink` trick.